spec/models/users_model_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+describe User, type: :model do
+  describe 'validation tests' do
+    let!(:user1){ create :user, email: 'dupa@dupa.pl' }
+    it 'ensure user is valid' do
+      expect(user1).to be_valid
+    end
+    context 'when name of user is too long' do
+      it 'rails validation error' do
+        user1.name = "a" * 51
+        expect(user1).to_not be_valid
+        expect(user1.errors.count).to eq 1
+        expect(user1.errors[:name]).to eq ["is too long (maximum is 50 characters)"]
+      end
+    end
+    context 'when name of user is not presence' do
```

Comment:
is not present. W sumie można ładniej te contexty podzielić, tzn context jeden od błędów związanych z nazwą i dwa "it", setup masz ten sam, dane zmieniasz już w uruchumionym teście, więc lepiej by wyglądało 
`context 'name errors'
i  

  it 'is blank' do
  end
  it 'is too short' do
  end
end `